fuel-core/src/lib.rs

```diff
@@ -2,6 +2,7 @@ pub use fuel_chain_config as chain_config;
 pub mod database;
 pub mod executor;
 pub mod model;
+mod query;
```

Comment:
Why is query located here? Wouldn't it be more suited to be directly in `/schema` since all of its associated functions are a part of a specific query?

Reply:
I have been discussing with @freesig and @xgreenx about moving the business logic of these queries out of the graphql handlers so that they can be unit tested and also allow us to support alternate entrypoints other than graphql in the future. Then the graphql handlers will have minimal logic to only accepting parameters and unwrapping types before calling our internal Rust API.

Reply:
I'm happy to move it down now